src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -155,4 +166,14 @@ public void validateCallbackErrorsAndWarnings(final CallbackResponse callbackRes
                 .withWarnings(callbackResponse.getWarnings());
         }
     }
+
+    private boolean logCallbackDetails(final String url) {
+        boolean logCallBack = false;
+        List<String> ccdCallbackLogControl = applicationParams.getCcdCallbackLogControl();
+        if (ccdCallbackLogControl.size() > 0 && (STAR_STR.equals(ccdCallbackLogControl.get(0))
+            || ccdCallbackLogControl.stream().anyMatch(url::contains))) {
```

Comment:
Just `return` this instead of initialising the boolean, setting in here and then returning..

Reply:
Could maybe be simplified slightly too?
```suggestion
        return ccdCallbackLogControl.contains(WILDCARD) || ccdCallbackLogControl.stream().anyMatch(url::contains);
```

Reply:
No. Not really. We would like to have the strict control definition of the wild card.